scripts/build-metamath

```diff
@@ -41,8 +41,8 @@ mkdir -p "$my_bindir"
 cp -p src/metamath "${my_bindir}"
 
 # If there's no more recent mmbiblio.html, use this one.
-if ! [ -e ../mmbiblio.html ] ; then
-  cp -p mmbiblio.html ../
-fi
+# if ! [ -e ../mmbiblio.html ] ; then
+#   cp -p mmbiblio.html ../
+# fi
```

Comment:
If it's not needed, then remove it instead of commenting it out.

Reply:
I'd lean towards removing it too, but don't consider this a blocker and am quite willing to defer to the choice that @david-a-wheeler will find more convenient in maintaining these scripts.